### PR TITLE
add goods selector

### PR DESCRIPTION
**Pull request by ly-manka**
[DEMO LINK](https://ly-manka.github.io/react_goods-selector-js/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -14,57 +15,75 @@ export const goods = [
...
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map((good, id) => {
```
Comment (etojeDenys) [inline src/App.jsx:41]:
```suggestion
          {goods.map((good) => {
```
it's an index, not the id

File: src/App.jsx [hunk 2]
```diff
@@ -14,57 +15,75 @@ export const goods = [
...
+            const isSelected = selectedGood === good;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+            return (
+              <tr
+                // eslint-disable-next-line react/no-array-index-key
+                key={id}
```
Comment (etojeDenys) [inline src/App.jsx:47]:
```suggestion
             
                key={good}
```

File: src/App.jsx [hunk 3]
```diff
@@ -14,57 +15,75 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState(goods.at(-2));
```
Comment (etojeDenys) [inline src/App.jsx:19]:
```suggestion
  const [selectedGood, setSelectedGood] = useState(goods.at(-2));
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]
few comments

**Review by nazarmatsevych — APPROVED** [review]
Good work!

**Pull request closed without merging**